.github/workflows/release.yaml

```diff
@@ -229,6 +231,7 @@ jobs:
           path: bound/kt/src/main/resources/
 
       - name: Deploy Release Version of Kotlin Project to Maven Central
+        if: ${{ env.SONATYPE_PASSWORD != '' }}
```

Comment:
we still need this?

Reply:
not really but it will be useful if we want to debug this in the future.

Reply:
Had to update some other stuff so I took this out too.